app/core/modules/Tasks/components/TaskHistory.vue

```diff
@@ -1,54 +1,144 @@
 <template>
-    <div class="history">
-        <div v-for="change in task.changes" :key="change.id" class="history-change">
-            <TeamAvatars class="history-change-avatar" :users="[change.user]" />
-
-            <template v-if="change.field === 'users'">
-                {{
-                    $t('projects.task_change_users', {
-                        user: change.user.full_name,
-                        date: fromNow(change.created_at),
-                        value:
-                            change.new_value && change.new_value.length
-                                ? JSON.parse(change.new_value)
-                                      .map(user => user.full_name)
-                                      .join(', ')
-                                : '',
-                    })
-                }}
-            </template>
-
-            <template v-else-if="change.field === 'status_id'">
-                {{
-                    $t('projects.task_change_to', {
-                        user: change.user.full_name,
-                        field: $t(`field.${change.field}`).toLocaleLowerCase(),
-                        value: getStatusName(change.new_value),
-                        date: fromNow(change.created_at),
-                    })
-                }}
-            </template>
-
-            <template v-else-if="change.field === 'priority_id'">
-                {{
-                    $t('projects.task_change_to', {
-                        user: change.user.full_name,
-                        field: $t(`field.${change.field}`).toLocaleLowerCase(),
-                        value: getPriorityName(change.new_value),
-                        date: fromNow(change.created_at),
-                    })
-                }}
-            </template>
-
-            <template v-else-if="change.field !== 'relative_position'">
-                {{
-                    $t('projects.task_change', {
-                        user: change.user.full_name,
-                        field: $t(`field.${change.field}`).toLocaleLowerCase(),
-                        date: fromNow(change.created_at),
-                    })
-                }}
-            </template>
+    <div>
+        <div ref="commentForm" class="comment-form">
```

Comment:
Why do you use a class here? This is only one element on the page with that class.

Reply:
I'm using the comment-form class for easy styling from line 371

Reply:
But this is only one object on that page and you should use ID then

Reply:
this is not a single element, this is a tree of elements, this code was not written by me, why it was written this way, I don’t know, I decided to leave it like this because my task was to combine components and not refactoring
![image](https://user-images.githubusercontent.com/51990904/227738395-09b885ff-184a-491c-aa13-ba85f37a6c85.png)
![image](https://user-images.githubusercontent.com/51990904/227738453-a37fe8e9-7ed0-4faf-9402-635f101e01dd.png)